pkg/mattermost/mattermost.go

```diff
@@ -232,6 +239,17 @@ func GenerateDeployment(mattermost *mattermostv1alpha1.ClusterInstallation, dbIn
 
 	envVarDB = append(envVarDB, masterDBEnvVar)
 
+	// Add init container to wait for DB setup job to complete
+	initContainers = append(initContainers, corev1.Container{
+		Name:            WaitForDBSetupContainerName,
+		Image:           "bitnami/kubectl:1.17",
+		ImagePullPolicy: corev1.PullIfNotPresent,
+		Command: []string{
+			"sh", "-c",
+			fmt.Sprintf("kubectl wait --for=condition=complete --timeout 5m job/%s", SetupJobName),
```

Comment:
Do you know how `kubectl wait` works under the covers? Is it polling the k8s api at a certain rate? My only concern would be if it polls every 30s then this could really slow down pod start-up

Reply:
I would assume it runs watch on the informer so whenever the job completes it finishes quite quickly. I will double-check to make sure.